src/services/tech-radar/client.ts

```diff
@@ -0,0 +1,16 @@
+interface TechRadarClient {
+  getTechRadarTechs(): Promise<RadarTech[]>;
```

Comment:
The name of this API is a bit redundant by `tech` twice 😅 what about just `getTechRadar`?